docs/installation.md

```diff
@@ -1,3 +1,13 @@
+# Quick install
+
+Run below command to install latest version of aeternity node.
+
+```bash
+wget https://raw.githubusercontent.com/aeternity/epoch/master/scripts/install.sh && chmod +x install.sh && ./install.sh 1.1.0
```

Comment:
Do I understand correctly that this is meant to be the same as the content of VERSION in the tree? If yes, please update it to 1.2.0.

Also, I see 1.1.0 below in this file while master has 1.2.0. Could you please rebase on current master?

Reply:
I tested with 1.1.0 and just forgot it .. fixed in 8dc3f4d5

Reply:
After https://www.pivotaltracker.com/story/show/162785706 I'll remove the version by default